### DISCUSSION TITLE
Supporting older browsers

### DISCUSSION THREAD

**Discussion by kisexu** [filamentphp/filament#8476] [Help]
The filament framework does not work properly in lower versions of the Chrome browser. 

![Snipaste_2023-09-14_12-08-20](https://github.com/filamentphp/filament/assets/1040190/0656a182-8009-421d-8462-5b6306d04e42)
![Snipaste_2023-09-14_12-09-57](https://github.com/filamentphp/filament/assets/1040190/57a7be60-ca54-46db-b3e4-98a7df727409)

After investigation, it was found that the frontend resources are using the JavaScript "Nullish coalescing operator" feature, which is not supported in Chrome versions below 80. 

I have researched a solution and found that you can use the plugins @babel/plugin-proposal-nullish-coalescing-operator and @babel/plugin-proposal-optional-chaining for transpiling. 

However, it seems that the filament documentation does not provide instructions on how to recompile the official frontend resources.

**Comment by leandrocfe** [comment]
They are compiled [here](https://github.com/filamentphp/filament/blob/3.x/bin/build.js)
Not sure if this section can help you [support/assets](https://filamentphp.com/docs/3.x/support/assets)
